src/matmul_kernels.py

```diff
@@ -0,0 +1,157 @@
+import torch
+import triton
+
+assert triton.__version__.startswith("2.1.0"), f"found triton {triton.__version__}, want 2.1.0*"
```

Comment:
consider replacing this with a warning. The kernel definitely works with 2.2.0, but it may be slower

Reply:
I'll just remove the kernel from this PR. It would be a matter for another PR.

Reply:
I brought it back